ansible_base/activitystream/views.py

```diff
@@ -14,7 +16,17 @@ class EntryReadOnlyViewSet(ReadOnlyModelViewSet, AnsibleBaseDjangoAppApiView):
 
     queryset = Entry.objects.all()
     serializer_class = EntrySerializer
-    permission_classes = [permissions.IsAuthenticated]
+
+    def get_permissions(self):
+        """
+        If the RBAC app is enabled, then we delegate to it to manage permissions.
+        Otherwise, we require superuser status.
+        """
+        if 'ansible_base.rbac' in settings.INSTALLED_APPS:
+            permission_classes = [permissions.IsAuthenticated]
```

Comment:
Let's also get an issue filed to make the required changes to queryset (or more likely, `filter_queryset`) to support this.

Reply:
#273